src/mobile/dropdown/index.stories.js

```diff
@@ -90,9 +91,10 @@ Base1.args = {
     {
       label: "Apple",
       value: "apple",
+      disabled: true,
     },
   ],
-  value: "orange",
+  value: "apple",
```

Comment:
Please don't modify the value, because it is used for checking the same values of item.
You can add a new case to check item.disabled or don't add. Both are OK.